README.md

```diff
@@ -1,2 +1,209 @@
 # mir_dataset_loaders
 common loaders for mir datasets
+
+
+## Examples
+
+### List available datasets
+```python
+import mir_dataset_loaders as mdl
+
+mdl.list_datasets()
+```
+
+### Download the Orchset Dataset
+```python
+import mir_dataset_loaders as mdl
+
+mdl.orchset.dowload()
```

Comment:
You should include something here about `data_home` and the `MIR_DATASETS_DIR` - very useful information that I otherwise had to piece together on my own, but should be available in the readme.